lib/oli/delivery/sections.ex

```diff
@@ -4385,11 +4385,18 @@ defmodule Oli.Delivery.Sections do
       on: ra.resource_id == rev.resource_id and ra.user_id == ^user_id,
       left_join: r_att in ResourceAttempt,
       on: r_att.resource_access_id == ra.id,
+      left_join: se in Oli.Delivery.Settings.StudentException,
+      on: se.resource_id == ra.resource_id and se.user_id == ^user_id,
```

Comment:
This must also constrain on `section_id`, otherwise a student that is enrolled in multiple course sections with the same resource_id will have incorrect results